automatic/docker-kitematic/tools/chocolateyInstall.ps1

```diff
@@ -21,5 +22,13 @@ foreach ($file in $files) {
 $desktop = $([System.Environment]::GetFolderPath([System.Environment+SpecialFolder]::DesktopDirectory))
 $link = Join-Path $desktop "Kitematic.lnk"
 if (!(Test-Path $link)) {
-    Install-ChocolateyShortcut -ShortcutFilePath "$link" -TargetPath "$toolsPath\Kitematic.exe" -WorkingDirectory "$toolsPath"
+    Install-ChocolateyShortcut -ShortcutFilePath "$link" -TargetPath $kitematicExe -WorkingDirectory "$toolsPath"
 }
+
+# Create sym-link for docker tray menu
```

Comment:
Add this in helper.ps1 as per 1.1.7

Reply:
Which part should become a function to go into a helper.ps1?